example/package.json

```diff
@@ -10,8 +10,16 @@
     },
     "./utils": "./dist/utils.mjs"
   },
-  "main": "./dist/module.cjs",
-  "types": "./dist/types.d.ts",
+  "main": "dist/module.mjs",
```

Comment:
I think anything that has the capability of running ESM will pick `exports`. What bug does this solve?

Reply:
so, we need to move it to ESM-only or patch cjs with `export.default` instead `module.exports`

Reply:
check also https://github.com/antfu/vite-plugin-inspect/issues/100